hazelcast-tpc-engine/src/test/java/com/hazelcast/internal/tpcengine/net/AsyncSocket_RpcTest.java

```diff
@@ -68,6 +69,9 @@ public abstract class AsyncSocket_RpcTest {
 
     public abstract ReactorBuilder newReactorBuilder();
 
+    protected void customizeSocketBuilder(AsyncSocketBuilder clientSocketBuilder) {
```

Comment:
Since this needs to be done for both client and server, I would call this the argument 'socketBuilder'